server/models/AuthenticationProvider.ts

```diff
@@ -97,7 +105,11 @@ class AuthenticationProvider extends Model {
     }
   }
 
-  disable = async (options?: SaveOptions<AuthenticationProvider>) => {
+  disable: (
+    options?: InstanceUpdateOptions<InferAttributes<AuthenticationProvider>>
+  ) => Promise<AuthenticationProvider> = async (
+    options?: InstanceUpdateOptions<InferAttributes<AuthenticationProvider>>
+  ) => {
```

Comment:
```suggestion
  disable: (
    options?: InstanceUpdateOptions<InferAttributes<AuthenticationProvider>>
  ) => Promise<AuthenticationProvider> = async (options) => {
```

No need for the duplication, TS can infer

Reply:
It can't, for some reason 🤔 

![Screenshot 2024-01-12 at 10 14 23](https://github.com/outline/outline/assets/15694566/d79b75c6-5499-48d2-8139-0ea4b5191d9f)

Reply:
Stupid me!